include/avif/avif.h

```diff
@@ -846,6 +846,8 @@ typedef struct avifImage
     // Owned by the avifImage and gets freed when calling avifImageDestroy().
     // gainMap->image->transformFlags is always AVIF_TRANSFORM_NONE.
     avifGainMap * gainMap;
+
+    // Version 1.2.0 ends here. Add any new members after this line.
```

Comment:
There are three extensible structs in avif.h: avifImage, avifDecoder, and avifEncoder.